StorageTests/TestSQLiteHistory.swift

```diff
@@ -1292,7 +1342,8 @@ class TestSQLiteHistory: XCTestCase {
 
             return history.getTopSitesWithLimit(20) >>== { topSites in
                 XCTAssertEqual(topSites.count, 20)
-                XCTAssertEqual(topSites[0]!.guid, "abc\(0)def")
+                XCTAssertEqual(topSites[0]!.guid, "abc\(5)def")
+                XCTAssertEqual(topSites[1]!.guid, "abc\(0)def")
```

Comment:
The only reason to fix this is that your test will break if SQLite returns things in a different order!